charts/gardener-extension-os-suse-chost/doc.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate ../../vendor/github.com/gardener/gardener/extensions/hack/generate-controller-registration.sh os-suse-jeos . ../../VERSION ../../example/controller-registration.yaml OperatingSystemConfig:suse-jeos OperatingSystemConfig:suse-chost
+//go:generate ../../vendor/github.com/gardener/gardener/extensions/hack/generate-controller-registration.sh os-suse-chost . ../../VERSION ../../example/controller-registration.yaml OperatingSystemConfig:suse-chost OperatingSystemConfig:suse-jeos
```

Comment:
Not at the scope of this PR, but it us very little change.

```suggestion
//go:generate ../../vendor/github.com/gardener/gardener/extensions/hack/generate-controller-registration.sh os-suse-chost . ../../VERSION ../../example/controller-registration.yaml OperatingSystemConfig:suse-chost OperatingSystemConfig:suse-jeos OperatingSystemConfig:memoryone-chost
```

PS. you have to run `make generate`